### ISSUE TITLE
Mercurial repository with git repo inside could be migrated but cannot be pushed to remote

### ISSUE THREAD

**Issue by stchar** [frej/fast-export#202]
If mercurial repository contains a git repository  hg-fast-import slurps such commits but 
git refuse to push it later:
```
$ git push -u origin --all
Enumerating objects: 2805, done.
Counting objects: 100% (2805/2805), done.
Delta compression using up to 12 threads
Compressing objects: 100% (840/840), done.
remote: error: object 0cd6549a1f221201f3506ff467b0980074430602: hasDotgit: contains '.git'
remote: fatal: fsck error in packed object
error: remote unpack failed: index-pack abnormal exit
```

I'm not sure if it is an issue with hg-fast-export, however any feature to filter out such files would be welcome

**Comment by chrisjbillington** [comment]
What version of `hg-fast-export` are you using?

This was probably fixed by #198 a few days ago.

**Comment by stchar** [comment]
Yep,  old version, before #198. Sorry, my bad
